### PR TITLE
Fix/7973 html display

### PR DESCRIPTION
**Pull request by whiskey**
## Description
This PR fixes the unreliable behaviour in the data privacy & usage screens by refactoring the HTML rendering in the app. The previously used `HtmlTextView` is replaced by `WKWebView` embedded in the layout (in the "App-Informationen" and the onboarding as well). 

Also the HTML has been sanitized and a gazillion inline CSS statements been refactored to use one (1) common .css file.

This refactoring reduces the memory footprint of the HTML rendered views, especially the data privacy view, from ~220MB to ~50MB.
html-before-after

## Link to Jira
[EXPOSUREAPP-7973](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7973)

## Screenshots
(all non-darkmode videos and screenshots are taken on an iPhone 6 running iOS 12.5.x)

https://user-images.githubusercontent.com/149976/122986419-b7180e80-d39f-11eb-90f0-edd1ee2df0c3.mp4

https://user-images.githubusercontent.com/149976/122986360-a2d41180-d39f-11eb-8b3f-99fb5174dce3.mp4

Screen Shot 2021-06-22 at 21 16 21
Screen Shot 2021-06-22 at 21 16 06

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Views/Common/HTMLView.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import WebKit
+
+final class HTMLView: WKWebView {
```
Comment (freshking) [inline src/xcode/ENA/ENA/Source/Views/Common/HTMLView.swift:8]: Great to see WKWebView in use for this.

**Review by freshking — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Views/Common/HTMLView.swift (hunk 1 above)
Comment (freshking) [reply src/xcode/ENA/ENA/Source/Views/Common/HTMLView.swift:8]: Sorry, this should have been a comment only.

**Review by NataliaLemmerth — COMMENTED** [review]
UA approved. No UI texts to review.

File: src/xcode/ENA/ENA/Source/Scenes/HTMLViewController/HTMLViewController.swift [hunk 2]
```diff
@@ -70,9 +72,38 @@ class HTMLViewController: UIViewController, DismissHandling {
 		}
 	}
 
-	@IBOutlet private weak var imageView: UIImageView!
-	@IBOutlet private weak var htmlTextView: HtmlTextView!
+    @IBOutlet private weak var containerView: UIView!
+    @IBOutlet private weak var imageView: UIImageView!
+	@IBOutlet private weak var htmlView: HTMLView!
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/HTMLViewController/HTMLViewController.swift:77]: indentation

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift [hunk 3]
```diff
@@ -6,7 +6,7 @@ import Foundation
 import UIKit
 
 class DynamicTableViewHtmlCell: UITableViewCell {
-	let textView = HtmlTextView()
+	let webView = HTMLView()
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift:9]: Will it work with dynamic tableview cell sizing?

**Review by KaiTeuber — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift (hunk 3 above)
Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift:9]: I tested the Onboarding which worked fine. Do you have something else in mind?

File: src/xcode/ENA/ENA/Source/Scenes/HTMLViewController/HTMLViewController.swift (hunk 2 above)
Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Scenes/HTMLViewController/HTMLViewController.swift:77]: Blame GitHub.
Screenshot 2021-06-23 at 13 37 11

**Comment by naveeddotio** [comment]
sehr geil 👍🏼

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift (hunk 3 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift:9]: Many Subviews inside App Information I guess

Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHtmlCell.swift:9]: Fixing the "Nutzungsbedingungen" and "Datenschutz" was the goal of this ticket and all cases (including onboarding) are shown in the video.

**Review by KaiTeuber — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**